### PR TITLE
Add statistical_method_for_oxide_growth problemt statement and solution

### PR DESCRIPTION
**Pull request by SecMeant**
Results are yet to be done and reported.

### CODE REVIEW

File: statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Problem statement
+We experience very poor quality of the oxide grown on the silicon wafers on both dry and wet thermal oxidation.  
+We observe the results in cases:
+
+1. Grown oxide does not provide a good dopant stop. We read N-type semiconductor in places that should have not been doped, because we grew 500nm thick oxide.
```
Comment (kornelu123) [inline statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md]: we read -> we observe

File: statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1. Grown oxide does not form an insulating gate, when we apply a silver glue based contact. The bigger the contact the more probability there is to obtain a leaking gate with resistivity dropping below 200k Ohms.
+1. We observe "cracks" and "islands" when looking at the grown oxide under SEM.
+
+# Proposed solution
+I suspect the poor quality of the oxide is due to contamination by particles from the air or chemical residue from the RCA clean, blocking a path to silicon to form a silicon dioxide during oxidation.  
+I suspect the contamination to follow a random distribution in space.  
+
+Therefore, I propose a statistical method, that grows the oxide in multiple stages, where each stage is loosely define as:
```
Comment (kornelu123) [inline statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md]: loosely define -> loosely defined

**Review by kornelu123 — COMMENTED** [review]
2 little comments, otherwise seems fine.

File: statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md (hunk 1 above)
Comment (SecMeant) [reply statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md]: How about `we read N-type semiconductor from the hot point probe test`? Which is what I wanted to write originally, but unconsciously ended up writing this crap.

Comment (SecMeant) [reply statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md]:
```diff
- from the hot point ...
+ during the hot point ...
```
?

Comment (kornelu123) [reply statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md]: yeah, that also sounds fine, maybe ``we read voltage indicating N-type ...``, but that may sound overcomplicated

Comment (kornelu123) [reply statistical_method_for_oxide_growth/statistical_method_for_oxide_growth.md]: > ```diff
> - from the hot point ...
> + during the hot point ...
> ```
> 
> ?

yeah, i think ``during`` sounds much better.

**Pull request merged**